resources/assets/js/components/modals/edit-songs-form.vue

```diff
@@ -57,9 +57,21 @@
                                 <label>Track</label>
                                 <input type="number" min="0" v-model="formData.track">
                             </div>
+                            <div class="form-row" style="display: none;">
+                                <label>Disc</label>
+                                <input type="number" min="0" v-model="formData.disc">
+                            </div>
+                            <div class="form-row" style="display: none;">
+                                <label>Year</label>
+                                <input type="number" min="0" v-model="formData.albumYear">
+                            </div>
+                            <div class="form-row" style="display: none;">
+                                <label>Genre</label>
+                                <input type="text" v-model="formData.genre">
+                            </div>
                         </div>
                         <div v-show="currentView === 'lyrics' && editSingle">
-                            <div class="form-row">
+                            <div class="form-row" style="display: none;">
```

Comment:
is there a reason for: style="display: none;" 
new text boxes and lyrics text box are hidden when editing songs